lab-matt/model/talent.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+const mongoose = require('mongoose');
+
+const talentSchema = mongoose.Schema({
+  talent: {type: String, required: true, unique: true},
+  description: {type: String, required:true},
+  weakness: {type: String},
+  userId: {type: mongoose.Schema.ObjectId, required: true}
```

Comment:
Not sure why there is a userId here. When a new resource is created, mongo adds an _id property with a random id number.